src/fuzz/fuzz-varlink-idl.c

```diff
@@ -19,7 +19,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 
         assert_se(str = memdup_suffix0(data, size));
 
-        r = sd_varlink_idl_parse(str, /* ret_line= */ NULL, /* column= */ NULL, &vi);
+        r = sd_varlink_idl_parse(str, /* reterr_line= */ NULL, /* column= */ NULL, &vi);
```

Comment:
Shouldn't the column get the same fix?

Reply:
Yes, curious that I didn't get an clang-tidy warning.